tests/collection-tests.yaml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Run tests on packaged collections
+  hosts: localhost
+  gather_facts: false
+  vars:
+    galaxy_requirements: "{{ playbook_dir | dirname }}/6/galaxy-requirements.yaml"
+    collections_path: /tmp/ansible_collections
+  tasks:
+    - name: Get list of installed packages to verify if podman is installed
+      package_facts:
+        manager: "auto"
+      # This is noisy (and potentially sensitive) to print on stdout
+      no_log: true
+
+    # This is so we can otherwise run unprivileged if podman is already installed
+    - when: "'podman' not in ansible_facts['packages'].keys()"
+      become: true
+      block:
+        - name: Install podman
+          package:
+            name: podman
+            state: present
+      rescue:
+        - name: Could not install podman
+          fail:
+            msg: |
+              Failed to elevate privileges and install podman.
+              Install podman manually or run ansible-playbook with elevated privileges.
+
+    - name: Install collections from galaxy
+      environment:
+        ANSIBLE_COLLECTIONS_PATH: "{{ collections_path }}"
+      command: ansible-galaxy collection install -r {{ galaxy_requirements }}
+      args:
+        creates: "{{ collections_path }}"
```

Comment:
Need to add a retry here because it can (and will) fail like this:

```
 ________________________________________   
< TASK [Install collections from galaxy] > 
 ----------------------------------------
        \   ^__^            
         \  (oo)\_______    
            (__)\       )\/\
                ||----w |                                                      
                ||     ||
                                                                               
fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["ansible-galaxy", "collection", "install", "-r", "/root/src/ansible-build-data/6/galaxy-requirements.yaml"], "delta": "0:00:28.981410", "end": "2022-05-17 19:47:26.388533", "msg": "non-zero return code", "rc": 1, "start": "2022-05-17 19:46:57.407123", "stderr": 
"ERROR! Error when getting the collection info for cloud.common from default (https://galaxy.ansible.com/api/) (HTTP Code: 502, Message: Bad Gateway Code: Unknown)", "stderr_lines": ["ERROR! Error when getting the collection info for cloud.common from default (https://galaxy.ansible.com/api/) (HTTP Code: 502, Message:
 Bad Gateway Code: Unknown)"], "stdout": "Starting galaxy collection install process\nProcess install dependency map", "stdout_lines": ["Starting galaxy collection install process", "Process install dependency map"]}
```

Reply:
Can use the same approach as in antsibull: https://github.com/ansible-community/antsibull/blob/e8595b5d1dfc4c77dc99fb95927270fcbaa7a40a/roles/build-release/tasks/build.yaml#L36-L48